src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,24 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class MainApp {
+    public static void main(String[] args) {
+
+        Machine truck = new Truck("Mikki");
+        Machine bulldozer = new Bulldozer("Eduardo");
+        Machine excavator = new Excavator("Maks");
+
+        Machine[] machines = {truck, bulldozer, excavator};
+        printTotal(machines);
+        System.out.println();
+    }
 
+    private static void printTotal(Machine[] machines) {
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
+        }
+        System.out.println(Arrays.toString(machines));
```

Comment:
The `System.out.println(Arrays.toString(machines));` does not provide an informative message about the machines. It's better to have messages that indicate what type of machine is working. Consider overriding the `toString` method in each `Machine` subclass to provide more informative output.